### PR TITLE
Doc/end to end test

### LINKED ISSUE #108: create a description of an end-to-end test
The purpose is to help the users to check saps engine, from submission via the dashboard (and API or DB) to archive.

We need to provide URLs to safe algorithms and known tasks to submit.

Also, it would be good to have a check of the output (both data and metadata, maybe the checksums?) And of the state of the catalog.

### PR DESCRIPTION
**Pull request by thiagoyeds**
Fix #108 and #136 

Link to NOPs versions:
`nop`
- [inputdownloading version](https://github.com/ufcg-lsd/saps-scripts-inputdownload/tree/nop)
- [preprocessing version](https://github.com/ufcg-lsd/saps-scripts-preprocessing/tree/nop)
- [processing version](https://github.com/ufcg-lsd/saps-scripts-processing/tree/nop)

`nop-fake-files`
- [inputdownloading version](https://github.com/ufcg-lsd/saps-scripts-inputdownload/tree/nop-fake-files)
- [preprocessing version](https://github.com/ufcg-lsd/saps-scripts-preprocessing/tree/nop-fake-files)
- [processing version](https://github.com/ufcg-lsd/saps-scripts-processing/tree/nop-fake-files)

`nop-download-files`
- inputdownloading version (developing)
- preprocessing version (developing)
- processing version (developing)

### CODE REVIEW

File: docs/end-to-end-test.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+# Configure
+
+It is necessary to configure the scripts that will be used by the SAPS pipeline to be able to produce the same result expected by the test, for that, we must configure three components to recognize these test scripts.
+
+**Note: After the changes, it is necessary to restart the components**
+
+### Configuring Dashboard
+
+In the [SAPS script file](/public/dashboardApp.js) replace the value of the variable `scriptsTags` by:
```
Comment (wesleymonte) [inline docs/end-to-end-test.md]: Broken link

File: docs/end-to-end-test.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      "docker_tag": "endtoend-test",
+      "docker_repository": "fogbow/worker"
+    }
+  ]
+}
+```
+
+### Configuring Dispatcher and Scheduler
+
+In the [SAPS script file](/resources/execution_scripts_tags.json) replace it with the following json:
```
Comment (wesleymonte) [inline docs/end-to-end-test.md]: Broken link

File: docs/end-to-end-test.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+# Test flow
+
+![Test flow](img/end-to-end-test-flow.png)
+
+1. User submits a task (new processing) to the Dashboard
+2. Dashboard forwards the processing request to the Dispatcher
+3. Dispatcher adds new processing information to the Catalog
+4. Scheduler retrieves the new information entered for processing
+5. Scheduler submits the task to a Job Scheduler Service that will process the scripts
```
Comment (wesleymonte) [inline docs/end-to-end-test.md]: Wouldn't it be a Job Execution Service?

File: docs/end-to-end-test.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+LC82150652015174LGN00_EVI.nc | b3d1592d0302edc8e115f14650f7e029
+LC82150652015174LGN00_G.nc | 9be610cf8e8b9a7ca2382a3c6c492b20
+LC82150652015174LGN00_LAI.nc | 1dac46d96353c901aa3fae7f2e831cc6
+LC82150652015174LGN00_NDVI.nc | aa1636bb24b03f43012b5940648d2de2
+LC82150652015174LGN00_Rn.nc | b75f223bcc956aee781e7e993f5fafcb
+LC82150652015174LGN00_SAVI.nc | 364b20834ca8640b098669ef0aa726e8
+LC82150652015174LGN00_TS.nc | 9b0d727c42d05f267c226eab1f1b1e0e
+
+
+After downloading all files, run the following command below for each file in the table above and compare the result with the md5sum column
```
Comment (wesleymonte) [inline docs/end-to-end-test.md]: I think it needs to be said that this checks the integrity of the file.

**Review by wesleymonte — CHANGES_REQUESTED** [review]

File: docs/end-to-end-test.md (hunk 1 above)
Comment (thiagoyeds) [reply docs/end-to-end-test.md]: Done

File: docs/end-to-end-test.md (hunk 2 above)
Comment (thiagoyeds) [reply docs/end-to-end-test.md]: Done

File: docs/end-to-end-test.md (hunk 3 above)
Comment (thiagoyeds) [reply docs/end-to-end-test.md]: Done, I made a mistake regarding the service

Comment (thiagoyeds) [reply docs/end-to-end-test.md]: I will update the image too ...

File: docs/end-to-end-test.md (hunk 4 above)
Comment (thiagoyeds) [reply docs/end-to-end-test.md]: Done

File: docs/end-to-end-test.md (hunk 3 above)
Comment (thiagoyeds) [reply docs/end-to-end-test.md]: Image updated

**Review by wesleymonte — APPROVED** [review]
Good job @thiagoyeds!

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**